packages/tailwindcss/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@formkit/tailwindcss",
+  "repository": "https://github.com/formkit/formkit",
+  "version": "1.0.0-beta.4",
+  "description": "A FormKit plugin for Tailwind that supplies custom variants based on input state",
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "contributors": [
```

Comment:
In here we're gonna want an export map that indicates which file should be loaded via "require" (the `cjs`) and which should be loaded via an `import` (the `mjs`). The other packages have this as a reference to checkout.